File: src/test/resources/languages/javascript/valid.js
// Valid JavaScript Sample
const PI = 3.14159;

class Sample {
    constructor(name) {
        this.name = name;
    }

    process() {
        console.log("Processing: " + this.name);
        return true;
    }
}

function topLevel() {
    const s = new Sample("test");
    s.process();
}

const arrow = (x) => x * 2;

File: src/test/resources/languages/java/valid.java
package com.example;

import org.junit.jupiter.api.Test;

import java.util.List;
import java.util.ArrayList;

/**
 * A sample Java class.
 */
public class Sample {
    private final String name;

    public Sample(String name) {
        this.name = name;
    }

    public void process(List<String> items) {
        for (String item : items) {
            System.out.println(name + " processes " + item);
        }
    }

    /**
     * The main method to run the sample.
     * @param args
     */
    @Test
    public static void main(String[] args) {
        Sample s = new Sample("Tester");
        List<String> data = new ArrayList<>();
        data.add("A");
        data.add("B");
        s.process(data);
    }
}